tsconfig.json

```diff
@@ -13,7 +13,7 @@
         "inlineSourceMap": true,
         "module": "commonjs",
         "outDir": "./dist",
-        "lib": ["es2016", "dom"],
+        "lib": ["es2016", "es2017", "dom"],
```

Comment:
@lchsam Why do we need es2017? It might be a bit "bleeding edge"...

Reply:
Functions like `Object.values` (which Jesse used in his solution for the 4th project) do not exist in ES2016. I thought it would be better to just use ES2017. Should I just switch back to ES2016 and check whether those functions exist first?

Reply:
Ah, I see. This is my concern: https://kangax.github.io/compat-table/esnext/

Reply:
I think ES2017 is stable. I would not worry about these functions disappearing.